test_settings.py

```diff
@@ -0,0 +1,2 @@
+DEBUG = True
```

Comment:
We should be able to get rid off the `run_tests` via similar changes to the settings file.

```suggestion
from workbench settings import *
from django.conf.global_settings import LOGGING

DEBUG = True

INSTALLED_APPS += ('wistiavideo', )
```